README.rst

```diff
@@ -71,6 +71,17 @@ Binaries for Python Extension Packages <http://www.lfd.uci.edu/~gohlke/pythonlib
 
 .. _Anaconda: https://www.continuum.io/downloads
 
+Error Reporting
+--------------
+
+In case of API usage errors SoundFile raises the usuall `ValueError` or `TypeError`.
+
+For other errors `soundfile.SoundFileError` is raised (used to be `RuntimeError`).
```

Comment:
After #273 is merged, you can use

    `SoundFileError`

instead of

    `soundfile.SoundFileError`

`` `RuntimeError` `` should also work, because I've added `intersphinx` in #273 as well.